index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
```

Comment:
please move this script tag position to before the body closing tag to make your code proffecional.

Reply:
- [ ] **[OPTIONAL]**
 Great adding the changes however, I think you could improve more on elaborating more on the changes your propose.
- Your reason here is not really to make the code professional, It should be in order for the browser to load the content of the page before executing the script. Kindly, work more on your reasons.